www/docs/resources.md

```diff
@@ -29,15 +34,16 @@ const service = {
 await runService(service);
 ```
 
-The service has two input collections `"users"` and `"groups"` (populated here with some initial data), no external dependencies (i.e. the service definition does not define the optional `externalServices` field), and one resource: `ActiveFriends`, defined as follows:
```

Comment:
Not sure I have understood the existing convention for the `"` marks, but they aren't in the source, so I removed them here.